Gems/AtomLyIntegration/CommonFeatures/Code/Source/PostProcess/FilmGrain/EditorFilmGrainComponent.cpp

```diff
@@ -0,0 +1,109 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#include <PostProcess/FilmGrain/EditorFilmGrainComponent.h>
+#include <AzCore/RTTI/BehaviorContext.h>
+
+namespace AZ
+{
+    namespace Render
+    {
+        void EditorFilmGrainComponent::Reflect(AZ::ReflectContext* context)
+        {
+            BaseClass::Reflect(context);
+
+            if (AZ::SerializeContext* serializeContext = azrtti_cast<AZ::SerializeContext*>(context))
+            {
+                serializeContext->Class<EditorFilmGrainComponent, BaseClass>()->Version(0);
+
+                if (AZ::EditContext* editContext = serializeContext->GetEditContext())
+                {
+                    editContext->Class<EditorFilmGrainComponent>("Film Grain", "Controls the Film Grain")
+                        ->ClassElement(Edit::ClassElements::EditorData, "")
+                        ->Attribute(Edit::Attributes::Category, "Graphics/PostFX")
+                        ->Attribute(
+                            AZ::Edit::Attributes::Icon, "Icons/Components/Component_Placeholder.svg") // [GFX TODO ATOM-2672][PostFX] need to create icons for PostProcessing.
```

Comment:
Lets not link **GFX TODO ATOM-2672** to new code. This is probably an old ticket. I would just say 
**// Create better icons for this effect.**